src/action.spec.ts

```diff
@@ -20,87 +15,66 @@ function createLabel(label: string) {
   };
 }
 
+jest.mock("@actions/core");
+jest.mock("@actions/github");
+
+let labels: Array<{ name: string }> = null;
+jest.mock("./fetchLabels", () => async () => Promise.resolve(labels));
+
 describe("check-versioning-action", () => {
   describe("parsing label information", () => {
-    it("parses major labels", () => {
-      github.context.payload = {
-        pull_request: {
-          number: 1,
-          labels: [createLabel("major")]
-        }
-      };
-
-      action();
+    afterEach(() => {
+      labels = null;
+      (core.setOutput as jest.Mock).mockReset();
```

Comment:
So this was the "magic" spell?

Reply:
No, the main change is that `action` is now `async`, so we have to have the test cases `async` as well.
This one is not needed, actually. I'll check and remove it.